src/components/TldrawCanvasLabelDialog.tsx

```diff
@@ -254,6 +262,7 @@ const LabelDialogAutocomplete = ({
           <div className="referenced-node-autocomplete">
             <Label>{referencedNode?.name}</Label>
             <AutocompleteInput
+              id="referenced-node-autocomplete-input"
```

Comment:
[nit] this could break with multiple canvas elements on the page

the "react"-y way is usually:
- create a `cont inputRef = useRef()`
- pass the ref to AutocompleteInput `ref={inputRef}`
- call `inputRef.current.focus()` in the `useEffect`

Now this is prohibited by `AutocompleteInput` not supporting forwarding refs (which we should eventually add, low priority and would be tricky). The workaround to that could be to pass the ref into the containing `div` and then do `div.getElementsByTagName("input")[0].focus()`